---
layout: default
title: Schedule
---


<div class="box">
          <h3>Keynote speakers</h3>

	  <!-- To be announced. -->
	  
          <table width="100%" border="1">
            <tbody>
              <tr>
                <td><img src="images/boeva.jpg" alt="Valentina Boeva" title="Valentina Boeva" /><br />
                </td>
                <td><a href="https://boevalab.inf.ethz.ch/index.html"> Valentina Boeva</a><br />
                  Assistant Professor<br />
                  ETH Zurich, Switzerland</td>
              </tr>
              <tr>
                <td><img src="images/goke.jpg" alt="Jonathan Göke" title="Jonathan Göke" /><br />
                </td>
                <td><a href="https://jglab.org/"> Jonathan Göke</a><br />
                  Group Leader<br />
                  Genome Institute of Singapore</td>
              </tr>
              <tr>
                <td><img src="images/marschall.jpg" alt="Tobias Marschall" title="Tobias Marschall" /><br />
                </td>
                <td><a href="https://marschall-lab.github.io/"> Tobias Marschall</a><br />
                  Professor<br />
                  Heinrich Heine University Düsseldorf, Germany</td>
              </tr>
            </tbody>
          </table>
          <p><br />
          </p>
	  

</div>

<h3>Wednesday July 23, 2025</h3>

<p>All times listed are in BST, Room: 01A</p>

<table style="width: 100%;" border="0">
<tbody>
<tr>
 <td style="vertical-align: top;"><strong>11:20-12:20</strong></td>
 <td style="vertical-align: top;">
<strong>Invited Presentation:</strong> Learning variant effects on chromatin accessibility and 3D structure without matched Hi-C data<br/>Presenter Valentina Boeva, ETH Zurich, Switzerland (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>12:20-12:40</strong></td>
 <td style="vertical-align: top;">Proceedings Presentation: Spatial transcriptomics deconvolution methods generalize well to spatial chromatin accessibility data<br/>Sarah Ouologuem, Laura D. Martens, Anna C. Schaar, Maiia Shulman, Julien Gagneur, Fabian J. Theis<br/>Presenter Laura D. Martens, Technical University Munich, Germany (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>12:40-13:00</strong></td>
 <td style="vertical-align: top;">Towards Personalized Epigenomics: Learning Shared Chromatin Landscapes and Joint De-Noising of Histone Modification Assays<br/>Tanmayee Narendra, Giovanni Visonà, Crhistian de Jesus Cardona, James Abbott, Gabriele Schweikert<br/>Presenter Tanmayee Narendra, University of Dundee, United Kingdom (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>14:00-14:20</strong></td>
 <td style="vertical-align: top;">Proceedings Presentation: Alevin-fry-atac enables rapid and memory frugal mapping of single-cell ATAC-Seq data using virtual colors for accurate genomic pseudoalignment<br/>Noor Pratap Singh, Jamshed Khan, Rob Patro<br/>Presenter Noor Pratap Singh, University of Maryland - College Park, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>14:20-14:40</strong></td>
 <td style="vertical-align: top;">Proceedings Presentation: Oarfish: Enhanced probabilistic modeling leads to improved accuracy in long read transcriptome quantification<br/>Zahra Zare Jousheghani, Noor Pratap Singh, Rob Patro<br/>Presenter Zahra Zare Jousheghani, University of Maryland, College Park, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>14:40-15:00</strong></td>
 <td style="vertical-align: top;">Identification of interactions defining 3D chromatin folding from micro to meso-scale<br/>Leonardo Morelli, Stefano Cretti, Davide Cittaro, Tiago P. Peixoto, Alessio Zippo<br/>Presenter Leonardo Morelli, University of Trento, Italy (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>15:00-15:20</strong></td>
 <td style="vertical-align: top;">SpliSync: Genomic language model-driven splice site correction of long RNA reads<br/>Wui Wang Lui, Liliana Florea<br/>Presenter Liliana Florea, Johns Hopkins University, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>15:20-15:30</strong></td>
 <td style="vertical-align: top;">adverSCarial: a toolkit for exposing classifier vulnerabilities in single-cell transcriptomics<br/>Ghislain Fievet, Julien Broséus, David Meyre, Sébastien Hergalant<br/>Presenter Ghislain Fievet, Université de Lorraine, France (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>15:30-15:40</strong></td>
 <td style="vertical-align: top;">Quality assessment of long read data in multisample lrRNA-seq experiments using SQANTI-reads<br/>Netanya Keil, Carolina Monzó, Lauren McIntyre, Ana Conesa<br/>Presenter Netanya Keil, University of Florida, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>15:40-16:00</strong></td>
 <td style="vertical-align: top;">Proceedings Presentation: Transcriptome Assembly at Single-Cell Resolution with Beaver<br/>Qian Shi, Qimin Zhang, Mingfu Shao<br/>Presenter Qian Shi, The Pennsylvania State University, United States (live stream)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>16:40-17:00</strong></td>
 <td style="vertical-align: top;"><strong>Invited Presentation:</strong> Bioinformatics analysis for long-read RNA sequencing: challenges and promises<br/>Presenter Elizabeth Tseng, Pacific Biosciences, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>17:00-18:00</strong></td>
 <td style="vertical-align: top;"><strong>Invited Presentation:</strong> Quantifying RNA Expression and Modifications using Long Read RNA-Seq<br/>Presenter Jonathan Göke, Genome Institute of Singapore (in person)</td>
</tr>
</tbody>
</table>

<h3>Thursday, July 24, 2025</h3>

<p>All times listed are in BST, Room: 01A</p>
            
<table style="width: 100%;" border="0">
<tbody>

<tr>
 <td style="vertical-align: top;"><strong>08:40-09:40</strong></td>
 <td style="vertical-align: top;"><strong>Invited Presentation:</strong> Pangenome based analysis of structural variation<br/>Presenter Tobias Marschall, Heinrich Heine University Düsseldorf, Germany (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>09:40-10:00</strong></td>
 <td style="vertical-align: top;"><strong>Invited Presentation:</strong> Resolving Paralogues and Multi-Copy Genes with Nanopore Long-Read Sequencing<br/>Presenter Sergey Nurk, Oxford Nanopore Technologies (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>11:20-11:40</strong></td>
  <td style="vertical-align: top;">Proceedings Presentation: CREMSA: Compressed Indexing of (Ultra) Large Multiple Sequence Alignments<br/>Mikaël Salson, Arthur Boddaert, Awa Bousso Gueye, Laurent Bulteau, Yohan Hernandez-Courbevoie, Camille Marchet, Nan Pan, Sebastian Will, Yann Ponty<br/>Presenter Mikaël Salson, CNRS, France (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>11:40-12:00</strong></td>
 <td style="vertical-align: top;">Proceedings Presentation: Exploiting uniqueness: seed-chain-extend alignment on elastic founder graphs<br/>Nicola Rizzo, Manuel Cáceres, Veli Mäkinen<br/>Presenter Nicola Rizzo, University of Helsinki, Finland (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>12:00-12:20</strong></td>
 <td style="vertical-align: top;">FroM Superstring to Indexing: a space-efficient index for unconstrained k-mer sets using the Masked Burrows-Wheeler Transform (MBWT)<br/>Ondřej Sladký, Pavel Veselý, Karel Brinda<br/>Presenter Ondřej Sladký, Charles University, Czechia (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>12:20-12:40</strong></td>
 <td style="vertical-align: top;">The Alice assembler: dramatically accelerating genome assembly with MSR sketching<br/>Roland Faure, Jean-François Flot, Dominique Lavenier<br/>Presenter Roland Faure, Institut Pasteur, Paris, France (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>12:40-13:00</strong></td>
 <td style="vertical-align: top;">BINSEQ: A Family of High-Performance Binary Formats for Nucleotide Sequences<br/>Noam Teyssier, Alexander Dobin<br/>Presenter Noam Teyssier, Arc Institute, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>14:00-14:20</strong></td>
 <td style="vertical-align: top;">Proceedings Presentation: Ultrafast and Ultralarge Multiple Sequence Alignments using TWILIGHT<br/>Yu-Hsiang Tseng, Sumit Walia, Yatish Turakhia<br/>Presenter Yu-Hsiang Tseng, University of California San Diego, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>14:20-14:40</strong></td>
  <td style="vertical-align: top;">Proceedings Presentation: GreedyMini: Generating low-density DNA minimizers<br/>Shay Golan, Ido Tziony, Matan Kraus, Yaron Orenstein, Arseny Shur<br/>Presenter Arseny Shur Bar Ilan University, Israel (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>14:40-15:00</strong></td>
 <td style="vertical-align: top;">Proceedings Presentation: LYCEUM: Learning to call copy number variants on low coverage ancient genomes<br/>Mehmet Alper Yilmaz, Ahmet Arda Ceylan, Gun Kaynar, A. Ercument Cicek<br/>Presenter Mehmet Alper Yilmaz, Bilkent University, Turkey (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>15:00-15:20</strong></td>
 <td style="vertical-align: top;">POPSICLE: a probabilistic method to capture uncertainty in single-cell copy-number calling<br/>Lucrezia Patruno, Sophia Chirrane, Simone Zaccaria<br/>Presenter Lucrezia Patruno, University College London Cancer Institute, Cancer Research UK Lung Cancer Centre of Excellence, United Kingdom (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>15:20-15:40</strong></td>
 <td style="vertical-align: top;">MutSuite: A Toolkit for Simulating and Evaluating Mutations in Aligned Sequencing Reads<br/>Kendell Clement<br/>Presenter Kendell Clement, University of Utah, United States (in person)</td>
</tr>
<tr>
 <td style="vertical-align: top;"><strong>15:40-16:00</strong></td>
 <td style="vertical-align: top;">Landscape of The Dark Genome’s variants and their influence on cancer<br/>Joao P. C. R. Mendonca, Kristoffer Staal Rohrberg, Peter Holst, Frederik Otzen Bagger<br/>Presenter Joao P. C. R. Mendonca, Rigshospitalet, Denmark (in person)</td>
</tr>
